tests/radv/test_radv_ipv6_ra.py

```diff
@@ -4,7 +4,12 @@
 
 import pytest
 
-from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory   # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import run_garp_service # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import run_icmp_responder # lgtm[py/unused-import]
+from tests.common.dualtor.dual_tor_mock import mock_server_base_ip_addr # lgtm[py/unused-import]
```

Comment:
I think this fixture is not used and not needed. This importing should can be removed.